src/components/Auth/Auth.jsx

```diff
@@ -0,0 +1,137 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import { Redirect } from 'react-router-dom';
+import TextField from 'material-ui/TextField';
+import Paper from 'material-ui/Paper';
+import RaisedButton from 'material-ui/RaisedButton';
+import { logIn } from '../../redux/actions';
+
+import { endPoints } from '../../config/routes';
+
+import './Auth.css';
+
+const user = {
+  email: 'ashutosh@hooli.com',
+  password: 'reactisawesome',
+};
+
+class Auth extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      email: '',
+      password: '',
+      emailErrorMessage: '',
+      passwordErrorMessage: '',
+      loggedIn: this.props.loggedIn,
+    };
```

Comment:
segregate state into fields and errors i.e it should be something like

this.state = {
  fields: {
     email: ''",
     password: '"',
  },
  fieldErrors: {}
}

note: fieldErrors should be generated dynamically with same name as field names.